x-pack/plugins/reporting/export_types/printable_pdf/server/execute_job/index.test.js

```diff
@@ -145,9 +176,9 @@ test(`adds forceNow to hash's query, if it exists`, async () => {
   const executeJob = executeJobFactory(mockServer);
   const forceNow = '2000-01-01T00:00:00.000Z';
 
-  await executeJob({ objects: [{ relativeUrl: 'app/kibana#/something' }], forceNow, headers: encryptedHeaders }, cancellationToken);
+  await executeJob({ objects: [{ relativeUrl: '/app/kibana#/something' }], forceNow, headers: encryptedHeaders }, cancellationToken);
```

Comment:
These relativeUrls had to be changed because switching to using `'server.basePath': '/sbp'` exacerbated an issue with the existing tests.